### PR TITLE
Add the Autumn shows for 2021

### PR DESCRIPTION
**Pull request by BenWoodford76**
Now we've got last year out of the way lets start with this year!!

### CODE REVIEW

File: _shows/19_20/cinnamon_tea.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+title: Cinnamon Tea
+playwright: Beth White
+student_written: true
+period: Spring
+season: Online
+season_sort: 390
+date_start: 2020-08-23
+date_end: 2020-08-23
+venue: Online (via YouTube)
```
Comment (samozzy) [inline _shows/19_20/cinnamon_tea.md:11]: We should standardise what our online venue is and update the rest accordingly.

File: _shows/19_20/i_think_he_was_called.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+season_sort: 410
+date_start: 2020-09-09
+date_end: 2020-09-09
+venue: Online (via YouTube)
+
+cast:
+- role: Fae
+  name: Sophie Curtis
+- role: Rosalind
+  name: Sodia Bassani
```
Comment (samozzy) [inline _shows/19_20/i_think_he_was_called.md:17]: *Sofia

File: _shows/19_20/cinnamon_tea.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    href: https://impactnottingham.com/2020/08/cinnamon-tea-nnt-quarantine-season/
+    snapshot: FBRbd
+    publisher: Impact
+    author: Pete Rouse
+    title: "‘CINNAMON TEA’ @ NNT QUARANTINE SEASON"
+    date: 2020-08-26
+    rating: 4/5
+    quote: "Overall ‘Cinnamon Tea’ is a beautiful exploration of the human condition and the emotions that we all feel, effectively framed in the digital age. It is something that I feel needs a repeat watch, to soak in every aspect of the psyche that is laid bare before us.
+
+"
```
Comment (samozzy) [inline _shows/19_20/cinnamon_tea.md:73]: Remove white space pls

File: _shows/19_20/julius_caesar.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+
+title: "[The Fall of] Julius Caesar"
+playwright: William Shakespeare
```
Comment (samozzy) [inline _shows/19_20/julius_caesar.md:4]: Presumably this was adapted by someone too?

File: _shows/19_20/spring_awakening.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+title: SPRING + AWAKENING
+playwright: Tara Anegada
+student_written: true
+period: Spring
+season: External
+season_sort: 320
+date_start: 2020-06-27
+date_end: 2020-06-28
+venue: New Theatre
```
Comment (samozzy) [inline _shows/19_20/spring_awakening.md:11]: Is this the correct venue?

**Review by samozzy — CHANGES_REQUESTED** [review]
So far so good!
Should the Upload festival be its own season a la IUDF? @BenWoodford76

File: _shows/19_20/i_think_he_was_called.md (hunk 2 above)
Comment (BenWoodford76) [reply _shows/19_20/i_think_he_was_called.md:17]: lol, I blame cold fingers

File: _shows/19_20/julius_caesar.md (hunk 4 above)
Comment (BenWoodford76) [reply _shows/19_20/julius_caesar.md:4]: that information is not on the spreadsheet

File: _shows/19_20/spring_awakening.md (hunk 5 above)
Comment (BenWoodford76) [reply _shows/19_20/spring_awakening.md:11]: probably should be online

File: _shows/19_20/spring.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  name: Katie Booth
+- role: Ernest Kollman
+  name: George Baxter
+- role: Mrs. Bergmann
+  name: Lily Bailes
+- role: Mr. Zundenschlag / Mr. Wedekind / Dr. Lemonade
+  name: Barney Hartwill
+
+crew:
+- role: Writer / Director
```
Comment (samozzy) [inline _shows/19_20/spring.md:29]: This can just be Director

File: _shows/19_20/cinnamon_tea.md (hunk 1 above)
Comment (BenWoodford76) [reply _shows/19_20/cinnamon_tea.md:11]: What are your thoughs on the name for this venue?

Comment (samozzy) [reply _shows/19_20/cinnamon_tea.md:11]: I wonder if we do 
```yml
venue: YouTube
venue_sort: Online
```
To leave it open for shows hosted elsewhere?

Comment (samozzy) [reply _shows/19_20/cinnamon_tea.md:11]: Further to this: where do the online TheSpace shows go? Under online or under TheSpace UK? 🤔

Comment (BenWoodford76) [reply _shows/19_20/cinnamon_tea.md:11]: I'd go TheSpaceUK

**Comment by samozzy** [comment]
Okie dokie, this is in good shape but there are still outstanding things. Some have content missing (Freshers' Fringe, Mushy Peas), some probably want removing from this PR and making into a new one if the info isn't readily available (An Evening With - though if we have the info let's just get it in). 

History Site Consistency:
- Venue sort discussion for online shows (see review comments)
- What do we do about end dates for online shows? I *think* we made a decision but can't remember what it was and am doing all this on my phone (document this if it isn't already!!)
- We can standardise and prettify the show YouTube links. Put them as an attribute for now and I'll do the templating in a separate PR.

These Shows:
- Digital Producer roles should probably just be producer - it's the show that was digital not the producer! 
- Edinburgh Showcase shows can have Edinburgh Showcase removed from the title (as presumably that is not the title of the show - but it does give a thought to a subtitle field for this kind of transfer...)

Question:
- Should Spring & Awakening exist twice as it's a "transfer" to Upload rather than a new show (more like NSDF than Edinburgh)? - Basically, how many times did this show exist and where? If it's Upload and Edinburgh that can be one entry; if it's NNT and Upload and Edinburgh that can be two, with Upload being part of the NNT one imo. Thoughts please!!

File: _shows/21_22/an_evening_with.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
+---
```
[the remark this answers is not in the document]
Comment (samozzy) [reply _shows/21_22/an_evening_with.md:1]: Resolved

**Comment by BenWoodford76** [comment]
@samozzy where exactly did you see a role of Digital Producer?

**Comment by samozzy** [comment]
@BenWoodford76 Spring, Awakening, and Spring and Awakening

**Comment by BenWoodford76** [comment]
Anything further from your list that needs addressing @samozzy ??

File: _shows/19_20/julius_caesar.md (hunk 4 above)
Comment (BenWoodford76) [reply _shows/19_20/julius_caesar.md:4]: Added

**Review by samozzy — APPROVED** [review]

**Pull request merged**